docs/osindex.html

```diff
@@ -42,6 +42,7 @@
     <div class='map-overlay-inner'>
         <button id='request'>Find localBuildings</button>
         <div>Total features: <span id='feature-count'></span></div>
+        <div>Total area <b>(metres squared)</b>: <span id='area'></span></div>
```

Comment:
also, what is total area ? (perhaps, "total buildings area" ?)

Reply:
also, should you use a number more human friendly ? (491138.964 not very friendly)

Reply:
Okay. What would you suggest? Would it be something like rounding to nearest whole number, and adding commas to every thousand, such as 491,138? Or even just rounding it to the nearest thousand squared metres?

Reply:
So it returns just 491

Reply:
Change unit ? Did u ask Rob?

Reply:
I'll ask him